lib/spaceGenerator.tsx

```diff
@@ -1,23 +1,24 @@
+import path from 'path';
+
 import SwaggerParser from "@apidevtools/swagger-parser";
 import { OpenAPI } from "openapi-types";
 import { basename } from "path";
```

Comment:
I think you could import `posix` from here instead of reimporting path

Reply:
Done. Thanks for the suggestion.